handbook/technical-overview/auth/integration-guidance.md

```diff
@@ -0,0 +1,54 @@
+# Tools and Examples
+
+Authentication is a security critical part of applications, SPs SHOULD implement it with high-quality OIDC libraries rather than implementing OIDC from scratch.
+
+## Tools
+
+We recommend these libraries and frameworks to integrate OIDC:
+
+**Authentication**
+
+- [oidc-client-js](https://www.npmjs.com/package/oidc-client) - OIDC implementation for client-side JavaScript applications.
+- [NextAuth.js](https://next-auth.js.org/) - Authentication for Next.js projects.
+- [node-openid-client](https://github.com/panva/node-openid-client) - OIDC implementation for Node.js.
+- [AppAuth](https://appauth.io/) - OIDC implementation for mobile apps: [iOS](https://github.com/openid/AppAuth-iOS), [Android](https://github.com/openid/AppAuth-Android), [JS](https://github.com/openid/AppAuth-JS), [React Native](https://formidable.com/open-source/react-native-app-auth/) and [Flutter](https://pub.dev/packages/flutter_appauth).
+- [IdentityModel.AspNetCore](https://github.com/IdentityModel/IdentityModel.AspNetCore) - OIDC implementation for .NET.
```

Comment:
For .NET think we should recommend the Microsoft.AspNetCore.Authentication.OpenIdConnect package, see https://docs.microsoft.com/en-us/dotnet/architecture/microservices/secure-net-microservices-web-applications/#authenticate-with-an-openid-connect-or-oauth-20-identity-provider